manifests/testing/file-host.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: cdi-file-host
+  labels:
+    cdi.kubevirt.io: ""
+    cdi.kubevirt.io/testing: ""
+spec:
+  selector:
+    matchLabels:
+      cdi.kubevirt.io/testing: ""
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        cdi.kubevirt.io/testing: ""
+    spec:
+      initContainers:
+      - name: init
+        image: kubevirt/cdi-func-test-file-host-init
```

Comment:
don't we need to make this manifest templated, so we pull from the right docker repo using the right tag? Otherwise when we deploy we may not be retrieving the artifact we just built locally.

Reply:
Agreed - the yaml has been mostly scratch to drive the file server.  Now that I've got the file generation and servers running, I'll templatize this file.

Reply:
Although that said, is this an image we want to publish or build on demand?  If we're going to publish it, then there are CI considerations as well that I can look at.